### PR TITLE
Add onChange prop to ReactVideoView

### PR DESCRIPTION
**Pull request by Pretzi**
## Description: 

This will add the onChange prop to ReactVideoView in order to get the native events from the native player (onVideoEnd, playerRemoved).

### Checklist for this pull request

* [✓ ] PR is scoped to one task
* [✓ ] Documentation included
* [ ] One of:
  * [ ✓] The PR is not making any UI change
  * [ ] The PR is making a UI change but not a product change

Thank you!

### CODE REVIEW

**Review by casajavi — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -82,7 +82,16 @@ import { APVideoPlayer } from 'react-native-zapp-bridge';
     },
   };
 
-  return <APVideoPlayer {...{ src, maxwidth, style }} />;
+  // Example
+  const onChange = event => {
+    event.persist();
+
+    if (event.nativeEvent.eventName === 'onVideoEnd') {
```
Comment (pintocarlos) [inline README.md:89]: Please document all of the possible eventNames available.

**Review by pintocarlos — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Pretzi) [reply README.md:89]: @pintocarlos This was done here https://github.com/applicaster/React-Native-Zapp-Bridge/pull/46/commits/1b2c495c0d11b66ec8394b29944361d0249ba4a9

**Comment by pintocarlos** [comment]
I feel that these should be listed properly in the MD format and not within code.
Let's please have a section that states the support of an `onChange` callback prop that is passed an event object and explain that this type of event could be of the different types mentioned.
Also, according to @casajavi this may vary by platform and by native player implementation. Let's make we are thorough with the details.

**Comment by pintocarlos** [comment]
@casajavi mentioned yesterday that this is not consistent depending on the player. Let's make sure we document that here. Thanks.

File: README.md (hunk 1 above)
Comment (casajavi) [reply README.md:89]: @Pretzi @pintocarlos I think for now this is fine. Once both iOS and Android have the protocols to start emitting more events then I will comeback and document that here.

**Review by casajavi — DISMISSED** [review]

**Comment by casajavi** [comment]
@Pretzi @pintocarlos I think for now this is fine. Once both iOS and Android have the protocols to start emitting more events then I will comeback and document that here.

**Review by pintocarlos — DISMISSED** [review]

**Review by pintocarlos — APPROVED** [review]

**Pull request merged**